services/121-service/src/notifications/whatsapp/whatsapp.service.ts

```diff
@@ -320,5 +324,32 @@ export class WhatsappService {
         );
       }
     }
+    if (
+      registrationsWithPendingMessage &&
+      registrationsWithPendingMessage.length > 0
+    ) {
+      this.sendPendingWhatsappMessages(registrationsWithPendingMessage);
+    }
+  }
+  private sendPendingWhatsappMessages(
+    registrationsWithPendingMessage: RegistrationEntity[],
+  ): void {
+    for (const registration of registrationsWithPendingMessage) {
+      if (registration.whatsappPendingMessages) {
+        for (const message of registration.whatsappPendingMessages) {
+          this.sendWhatsapp(
+            message.body,
+            message.to,
+            message.messageType
+              ? (message.messageType as IntersolvePayoutStatus)
+              : null,
+            message.mediaUrl,
+            message.registrationId,
+          ).then(() => {
+            this.pendingMessageRepo.remove(message);
```

Comment:
@RubenGeo is it indeed fine to remove instead of save and flag as sent? (Just like voucher). I guess we still save the message in the twilio-message table. So fine I guess.

Reply:
Yes, this was also my thought process